From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: nooga <xnooga@gmail.com>
Date: Sat, 15 May 2021 01:21:59 +0200
Subject: [PATCH] Install the game data to /usr/local

---
 config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.mk b/config.mk
index 6f14b53a4e8586de1fccb16b5b2766b7e67cc4ff..02a53a5730ee72400c78f85e357d804cccb4bdbd 100644
--- a/config.mk
+++ b/config.mk
@@ -1,5 +1,5 @@
 # Where to look for game data files (found in 'bin'). Must be without trailing slashes!
-DATADIR := .
+DATADIR := /usr/local/share/games/brogue
 
 # Host and target platform. Values: LINUXLIKE, WINDOWS, OS2
 SYSTEM := LINUXLIKE
